optimism/src/mips/witness.rs

```diff
@@ -22,7 +22,7 @@ pub const NUM_DECODING_LOOKUP_TERMS: usize = 2;
 pub const NUM_INSTRUCTION_LOOKUP_TERMS: usize = 5;
 pub const NUM_LOOKUP_TERMS: usize =
     NUM_GLOBAL_LOOKUP_TERMS + NUM_DECODING_LOOKUP_TERMS + NUM_INSTRUCTION_LOOKUP_TERMS;
-pub const SCRATCH_SIZE: usize = 38;
+pub const SCRATCH_SIZE: usize = 60; // TODO: Delete and use a vector instead
```

Comment:
I will approve this change, without testing though as it does not imply any security/functional issues. I suppose it is by running the code you get this value.